datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -104,7 +104,17 @@ def create(ctx, name, integration_type, location, non_interactive, quiet, dry_ru
     root = resolve_path(location) if location else get_root()
     path_sep = os.path.sep
 
-    integration_dir = os.path.join(root, normalize_package_name(name))
+    if integration_type == 'snmp_tile' and name.startswith('snmp_'):
```

Comment:
I think there may be a bug here, when we create intg's we want users to use the docs version of the name/display name like `SNMP Cisco` instead of `snmp_cisco`. I tested by creating a tile for `SNMP Cisco` and it generated `snmp_snmp_cisco`. 
 
 ```
 $ ddev create "SNMP CCisco" -t snmp_tile
Created in `/Users/christine.chen/dev/integrations-core`:
snmp_snmp_ccisco
├── assets
│   └── service_checks.json
├── README.md
└── manifest.json
```

Reply:
Docs name in that case would be Cisco, not SNMP Cisco (as per discussion with product). But I see your point in having snmp has prefix in a name with spaces, which is a case I didn't consider. I would recommend opening a ticket to handle other cases, and merge this as a first version. Let me know if you think it should be changed in this version.

Reply:
I could use the `normalized_name` here to check the name, but it would be more challenging to propose an alternative name. And I'm not sure it's worth the effort, as the tool would be used internally.

Reply:
Ah okay, since this option is available to any contributor using ddev, we sometimes to get contributions and this leads to more difficulty converting all the proper values in manifest.json in terms of docs/tiles after creating scaffolding. Should use `snmp` instead of `snmp_tile` to avoid confusion with `tile` type?

Reply:
not totally comfortable with only `snmp`: this command creates a new tile for `snmp`. It's true that in the future we might move `profile` and `dashboard`. What about `snmp_vendor` ?